src/components/atoms/Input/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from "react";
+
+interface IInput {
+  type: string;
+  name: string;
+  value?: string | number;
+  ref?: React.Ref<HTMLInputElement>;
+  placeholder?: string;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  padding?: "none" | "medium";
+  color?: "default" | "transparent";
+  border?: boolean;
+}
+
+const Input: FC<IInput> = ({
+  type,
+  name,
+  value,
+  ref,
+  placeholder,
+  onChange,
+  padding,
+  color,
+  border,
+}) => {
+  const inputPadding = padding === "medium" ? "p-4" : "p-0";
```

Comment:
what in case u have more differcified paddings?
the thing that you split by multiole variable does not solve an issue 
it is complicated to read
so better do function